Essentials/src/com/earth2me/essentials/commands/Commandmail.java

```diff
@@ -32,7 +32,7 @@ public void run(final Server server, final User user, final String commandLabel,
             final List<String> mail = user.getMails();
             if (mail.isEmpty()) {
                 user.sendMessage(tl("noMail"));
-                throw new NoChargeException();
+                return;
```

Comment:
Turns out I'm completely wrong and this should have not been changed. Same with line 92 - it needs the throw new NoChargeException() to avoid erroneously charging the user